pages/about.html

```diff
@@ -4,9 +4,62 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link rel="stylesheet" href="style.css">
+  <title>About me</title>
 </head>
 <body>
-  
+  <div class="container">
+    <header>
+      <h1>My life </h1>
+    <nav>
+        <ul>
+          <div class="box">
```

Comment:
The `<div>` element cannot be a child of the `<ul>` tag. Remember to run your HTML through the HTML validator!